runtime/oti/j9nonbuilder.h

```diff
@@ -5653,6 +5712,12 @@ typedef struct J9JavaVM {
 	omrthread_monitor_t jclCacheMutex;
 	UDATA arrayletLeafSize;
 	UDATA arrayletLeafLogSize;
+	UDATA contiguousHeaderSize;
+	UDATA discontiguousHeaderSize;
+#if defined(J9VM_ENV_DATA64)
+	BOOLEAN isDataAddrPresent;
```

Comment:
@dmitripivkine do you think we should add 'indexable' in any/all of these?

contiguousIndexableHeaderSize
discontiguousIndexableHeaderSize
isIndexableDataAddrPresent

Reply:
yes, I think this is good idea

Reply:
When we cache them, depending of the context (class or instance name), we may or may not use 'indexable'.
 
For example, class ArrayletObjectModel or instance indexableObjectModel give clear context, so the cached variables would not really need 'indexable'.

On the other side ObjectAllocationAPI deals with both indexable and non-indexable object allocations, so I'd use 'indexable' there, to avoid possible confusion what header it refers too (albeit, the code is all inlined and it can be found out with not much effort).